lms/static/scripts/frontend_apps/config.js

```diff
@@ -4,4 +4,4 @@ import { createContext } from 'preact';
  * Configuration object for the file picker application, read from a JSON
  * script tag injected into the page by the backend.
  */
-export const Config = createContext({});
+export const Config = createContext({ api: {} });
```

Comment:
What did this break when it wasn't included? Do we have to do this for a lot of other stuff?